custom_components/mypyllant/const.py

```diff
@@ -16,6 +16,7 @@
 API_DOWN_PAUSE_INTERVAL = 15 * 60  # in seconds
 
 SERVICE_SET_QUICK_VETO = "set_quick_veto"
+SERVICE_SET_QUICK_VETO_AMBISENSE = "set_quick_veto_ambisense"
```

Comment:
I don't think it is used.

Moreover, do you want a separate service to handle quick veto for ambisense ? From user point of view, I think it is clearer to only have one service and the integration is dealing with the technical aspect

Reply:
Yeah, I came to the same realization half-way through and deleted the half-finished service, but left the constant in by accident. Good catch!